azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/state/ParticipantsState.kt

```diff
@@ -13,4 +13,5 @@ internal data class ParticipantsState(
     val participantsReadReceiptMap: Map<String, OffsetDateTime>,
     val latestReadMessageTimestamp: OffsetDateTime,
     val localParticipantInfoModel: LocalParticipantInfoModel,
+    val hiddenParticipant: Set<String>,
```

Comment:
iOS side uses maskedParticipant across for consistency. Not sure whether we need to align with this naming.